### PR TITLE
Adding a first version for the terminology page!

### PR DESCRIPTION
**Pull request by alix-tz**
See #52!

### CODE REVIEW

**Comment by alix-tz** [comment]
Warning, there may be conflicts to fix between this branch and fix/#23 on the index.md page!

File: docs/index.md [hunk 1]
```diff
@@ -15,22 +15,26 @@ You can find more information about eScriptorium and the context of its producti
...
 The [**Quick-start**](quick-start.md) is intended as a tutorial to introduce completely new users to the basics of using eScriptorium. It should be envisioned as a gateway to the walkthrough section which is much more detailed.
 
-## FAQ
+## Walkthrough 🚶🏽
 
-The [**FAQ**](FAQ.md) section covers frequent questions around the application and the documentation.
+Covering every features in an application like eScriptorium produces a long documentation, so we have broken it down into several pages in the "Walkthrough" section.
 
-## Walkthrough
+For example, [**Import data**](import.md) covers feature related to importing images, transcriptions or models in the application. The [**Automatic prediction**](predict.md) section covers features related to using models to predict the lay out (segmentation) or the transcription of a document.
```
Comment (FloChiff) [inline docs/index.md:26]: the first "feature" should have an "s"

File: docs/terminology.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- A *document-part* is simply an image (a digitization) of a document: it can be the photography of a double page, a single page, or even cropped portion of such images. Sometimes it is called an "Element".  
+- A *document* is a collection of such images, similar to how you would organize a directory containing images on your file system.
+- *Projects* can be understood as one more level to gather documents (themselves groups of images), but they can be envisioned as workspaces for a group of users working for the same project. For example, [tags](tags.md) are managed at project level, available for all the documents inside the given project.
+
+## Edition panels
+
+The edition panels refer to the panels (5) available in the edition page. This page is accessed via the following URL: `{base_url}/document/{doc_id}/part/{part_id}/edit/` (where `base_url` is the URL of the server where the application is deployed, `doc_id` the id of your document, and `part_id` the id of the document part being edited).
+
+There are multiples wats to access the edition page, for example:
```
Comment (FloChiff) [inline docs/terminology.md:23]: "ways" not "wats"

File: docs/terminology.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - [some of the transcription features](transcribe.md#editing-with-the-transcription-panel)
+
+5. **Text** (++control+5++), used for:
+    - [other transcription features](transcribe.md#editing-with-the-text-panel)
+    - adding [textual annotations](annotate.md#text-annotations)
+    - changing the [order of the lines](segment.md#reordering)
+
+
+!!! Tip
+    You can toggle as many panel as you want at once. It is not advised to have all of them active at the same time, though, because it limits the readability of the documents and you may unintentionally call some shortcuts.
```
Comment (FloChiff) [inline docs/terminology.md:54]: "panel" should take an "s" ?

**Comment by alix-tz** [comment]
Hop! Ready for merge!

**Pull request merged**